config/crd/bases/crdb.cockroachlabs.com_crdbclusters.yaml

```diff
@@ -255,6 +255,12 @@ spec:
                           type: string
                       type: object
                   type: object
+                supportsAutoResize:
```

Comment:
@chrislovecnm the flag name can be confusing.
I mean from code I see it just does a rolling restart of sts if it's true. But actual resize of PVC is supported by default, independent of this flag. I think this can be confusing, can we rename it to something more closer to its implementation like `restartPodsOnResize`?

Reply:
I am not talented with naming, but we need something about that the PVC supports pods on restart. Names are hard 😆 

@chrisseto PRTAL

Reply:
`restartPodsOnResize` sounds great!

How about ''Set to true if ExpandInUsePersistentVolumes is enabled on kubelet; volumes will be resized without pod disruption" as the description?

Reply:
> How about ''Set to true if ExpandInUsePersistentVolumes is enabled on kubelet

but that would mean host access to kubelet right? I'm not aware that kubernetes API exposes the enabled feature gates